src/js/peteshow-controller.coffee

```diff
@@ -1,5 +1,6 @@
-_  = require('lodash')
-cs = require('calmsoul')
+_     = require('lodash')
+cs    = require('calmsoul')
+store = require('./peteshow-storage')
```

Comment:
I was going to start working on the storage stuff, but decided to hold off until working with @brousalis ... there's really no need to pull it in this way, we could grab it off of Peteshow.store.